spec/models/edition_spec.rb

```diff
@@ -207,4 +207,40 @@
       end
     end
   end
+
+  describe "#assign_access_limit" do
+    it "can set an access limit" do
+      edition = build(:edition)
+      user = build(:user)
+
+      expect { edition.assign_access_limit(limit_type: :all_organisations, user: user) }
+        .to change { edition.access_limit }
+        .from(nil)
+        .to(an_instance_of(AccessLimit))
+
+      expect(edition.access_limit).to be_all_organisations
+      expect(edition.access_limit.created_by).to eq(user)
+      expect(edition.access_limit.revision_at_creation).to eq(edition.revision)
+    end
+
+    it "can remove an access limit" do
+      edition = build(:edition, :access_limited)
+
+      expect { edition.assign_access_limit(limit_type: nil, user: build(:user)) }
```

Comment:
This seems a bit counter-intuitive, depending on how you interpret 'nil'. Perhaps a dedicated `remove_access_limit` method would be clearer, or an explicit 'everyone' option?

Reply:
Did you read PR description?

Reply:
Indeed I did - and the commit message that invites some comments about it ;-). But it's quite a minor thing, really, so I'm not fussed.

Reply:
After much pondering I've gone with remove_access_limit and just dealt with it not specifying assign. It had the advantage that I could remove the keyword arguments which seemed inconsistent with most similar methods.